### PR TITLE
Run `brew style --fix`

### PR DESCRIPTION
**Pull request by MikeMcQuaid**
Needed for https://github.com/Homebrew/brew/pull/13477

Includes a few manual fixes.

### CODE REVIEW

File: Formula/gspell.rb [hunk 1]
```diff
@@ -129,7 +129,7 @@ def install
 	$(BUILT_SOURCES)
 
  libgspell_core_la_LIBADD =	\
-+	$(GTK_MAC_LIBS)		\
++	$(GTK_MAC_LIBS) \
```
Comment (Bo98) [inline Formula/gspell.rb:132]: This is a false positive but won't beak anything in this case.

File: Formula/lua@5.1.rb [hunk 2]
```diff
@@ -160,7 +160,7 @@ def caveats
 -LUA_A=	liblua.a
 +LUA_A=	liblua.5.1.5.dylib
  CORE_O=	lapi.o lcode.o ldebug.o ldo.o ldump.o lfunc.o lgc.o llex.o lmem.o \
-	lobject.o lopcodes.o lparser.o lstate.o lstring.o ltable.o ltm.o  \
+	lobject.o lopcodes.o lparser.o lstate.o lstring.o ltable.o ltm.o \
```
Comment (Bo98) [inline Formula/lua@5.1.rb:163]: This is a false positive and will break this patch.

File: Formula/sratoolkit.rb [hunk 3]
```diff
@@ -142,11 +142,11 @@ def install
...
-                 ${CMAKE_INSTALL_PREFIX}/bin         \
--                ${CMAKE_INSTALL_PREFIX}/lib64       \
-+                ${CMAKE_INSTALL_LIBDIR}             \
-                 ${CMAKE_SOURCE_DIR}/shared/kfgsums  \
+                 ${CMAKE_INSTALL_PREFIX}/bin/ncbi \
+                 /etc/ncbi \
+                 ${CMAKE_INSTALL_PREFIX}/bin \
+-                ${CMAKE_INSTALL_PREFIX}/lib64 \
++                ${CMAKE_INSTALL_LIBDIR} \
+                 ${CMAKE_SOURCE_DIR}/shared/kfgsums \
```
Comment (Bo98) [inline Formula/sratoolkit.rb:150]: This is a false positive and will also break this patch

File: Formula/lua@5.1.rb (hunk 2 above)
Comment (MikeMcQuaid) [reply Formula/lua@5.1.rb]: @bo98 Removed this commit in latest push

File: Formula/sratoolkit.rb (hunk 3 above)
Comment (chenrui333) [reply Formula/sratoolkit.rb]: if it is not `long build`, we can also do `ci-no-bottles` for the PR run to see if the patch would be broken.

Comment (MikeMcQuaid) [reply Formula/sratoolkit.rb]: @chenrui333 I'd rather just remove the commit for now.

**Review by chenrui333 — COMMENTED** [review]
LGTM

**Review by Bo98 — APPROVED** [review]

**Pull request merged**